tree-sitter-topas/grammar.js

```diff
@@ -153,14 +170,17 @@ module.exports = grammar({
       ')',
     ),
 
-    _block_item: $ => choice(
+    _block_item: $ => prec.right(choice(
       $.definition,
-      $.equation,
       $._global_preprocessor_directive,
       $._expression,
       $.refined_parameter,
       $.unrefined_parameter,
-    ),
+      $.initialisation_equation,
+      $.reassignment_equation,
```

Comment:
Why not `$._equation`?

Reply:
I did this because currently `$._equation` requires an `$.identifier` at the start, but `$._block_item`, which is used to make up the body of macros or directives,  can contain just about anything - including a standalone equation, without the `$.identifier`. I'm trying to strike a balance between creating a sensible structure tree within these bodies while allowing the user as much freedom as possible.

It will still look the same on the syntax tree since `$._block_item` is repeated and can be an `$.identifier`. 

I plan to get rid of `$._equation` entirely in the next PR, as it isn't a natural structure within TOPAS, but it still needs to be there for this one or the tests will fail.